src/app/redux/hooks.ts

```diff
@@ -0,0 +1,7 @@
+"use client";
```

Comment:
You don't need the use client here. This is a `.ts` file which only contains functions. The use client directive is only used in a `.tsx`  file that needs to run on the client's device rather than server side. You can get rid of that